plugins/askrene/askrene.c

```diff
@@ -764,12 +766,14 @@ static struct command_result *json_getroutes(struct command *cmd,
 	 */
 	/* FIXME: Typo in spec for CLTV in descripton! But it breaks our spelling check, so we omit it above */
 	const u32 maxdelay_allowed = 2016;
+	const u32 default_maxparts = 6;
```

Comment:
I think this should be increased (or default to infinite) and let xpay handle it.  Eventually it should try to detect last-hop failures and maybe reduce in that case?